packages/algob/src/internal/deployer.ts

```diff
@@ -142,7 +142,25 @@ class DeployerBasicMode {
   }
 
   /**
-   * Description: Opt-In to ASA for a single account. The opt-in transaction is
+   * Appends signature (using signer's sk) to multi-signed logic signature. If multisig is not found
+   * then new multisig is created
+   * eg. appending own signature to a signed lsig (received from multisignature account address network)
+   * @param lsig Logic Sig object
+   * @param signer: Signer Account which will sign the smart contract
+   * @param mparams: passed when signing a new multisig
+   * @returns multi signed logic signature (with appended signature from signer's sk)
+   */
+  signMultiSig (lsig: LogicSig, signer: rtypes.Account, mparams?: MultisigMetadata): LogicSig {
```

Comment:
This function only signs stateless smart contracts. We should change this name to something more appropriate. 

This name can be misleading, as we might be adding a similar function for general transactions too.

Reply:
Could you please suggest something?

Reply:
maybe `signLogicSigMultiSig` or `signLsigMultiSig`

Reply:
`signLogicSigMultiSig` works